Sources/MongoSwift/ConnectionPool.swift

```diff
@@ -97,29 +100,33 @@ internal class ConnectionPool {
         return try body(connection)
     }
 
-    /// Temporary API for testing purposes. This will likely change in SWIFT-471.
-    internal func setSSLOpts(caFile: String?, pemFile: String?, allowInvalidHostnames: Bool? = nil) throws {
-        // we need to use asCString rather than withCString to avoid nesting
-        // a ton of closures and handling every permutation of set/unset options.
-        let caString = caFile?.asCString
-        let pemString = pemFile?.asCString
+    /// Sets TLS/SSL options that the user passes in through the client level.
+    private func setTLSOptions(_ options: TLSOptions) throws {
```

Comment:
I think it might be worth calling out in this docstring for our own future reference that you must call this from the init before the pool has been used